pkg/columns/formatter/json/json.go

```diff
@@ -157,18 +156,18 @@ func (f *Formatter[T]) getPrinter(level int, cols []*column[T]) func(entry *T, b
 	// Better name?
 	handlePrefix := func() {
 		if !first {
-			funcs = append(funcs, func(entry *T, buf *encodeState) {
+			funcs = append(funcs, func(indent string, entry *T, buf *encodeState) {
 				buf.WriteByte(',')
 				if f.options.prettyPrint {
-					buf.WriteString("\n" + indent)
+					buf.WriteString("\n" + indent + "  ")
```

Comment:
Fyi: in isolated tests, calling this separately (`WriteByte('\n')`, `WriteString(indent)`, `Write(LevelIndent)`, where `LevelIndent = []byte("  ")` performed 3x better than the string concatenation; however, in the full scenario this didn't really have weight.

Reply:
That didn't make any difference for me in any test I did. The difference was only in concatenating the string before writing.

Reply:
I updated the code to avoid the string concatenation. It was not difficult so why not?